docs/process/labels.md

```diff
@@ -3,40 +3,47 @@
 This section outlines the labels currently used in the project, and adds context
 about what each label represents.
 
-## Common labels
+## Areas of Work
```

Comment:
Areas is the biggest change here, and it feels like the best way to group together issues related to a specific category, while giving us the flexibility to create new labels or deprecate them down the track if they are unused.

Are there any other areas you can think of that we might list here?